bin/git-up

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+git fetch origin
+git rebase origin/master
```

Comment:
```
       -r, --rebase[=false|true|preserve]
           When true, rebase the current branch on top of the upstream branch after fetching. If there is a remote-tracking branch corresponding to the upstream branch and the upstream branch was rebased
           since last fetched, the rebase uses that information to avoid rebasing non-local changes.
```